core/kits/assets/js/globals/global-select-behavior.js

```diff
@@ -233,14 +233,17 @@ export default class GlobalControlSelect extends Marionette.Behavior {
 
 	printGlobalToggleButton() {
 		const $globalToggleButton = jQuery( '<div>', { class: this.getClassNames().popoverToggle + ' elementor-control-unit-1' } ),
-			$globalPopoverToggleIcon = jQuery( '<i>', { class: 'eicon-globe' } );
+			$globalPopoverToggleIcon = jQuery( '<i>', { class: 'eicon-globe' } ),
+			$globalsLoadingSpinner = jQuery( '<span>', { class: 'elementor-control-spinner' } )
+				.html( '<i class="eicon-spinner eicon-animation-spin"></i>&nbsp;</span>' );
```

Comment:
```suggestion
				.html( '<i class="eicon-spinner eicon-animation-spin"></i></span>' );
```

Reply:
@matipojo I replaced the space with a CSS margin. The space character actually gave a margin from the globals toggle.